ggml.c

```diff
@@ -9311,7 +9439,7 @@ void ggml_graph_compute(struct ggml_context * ctx, struct ggml_cgraph * cgraph)
     const int n_threads = cgraph->n_threads;
 
     struct ggml_compute_state_shared state_shared = {
-        /*.spin      =*/ GGML_LOCK_INITIALIZER,
+        /*.lock      =*/ {},
```

Comment:
{0}?

Reply:
I'm used to `{}` doing zero-initialization in C++. Is `{0}` needed here?

Reply:
Yes, VS2002 complains about it.